src/WinRT.Runtime/AgileReference.cs

```diff
@@ -59,9 +80,9 @@ internal unsafe AgileReference(IntPtr thisPtr)
         }
 
 
-        public IObjectReference Get() => _cookie == IntPtr.Zero ? ABI.WinRT.Interop.IAgileReferenceMethods.Resolve(_agileReference, IUnknownVftbl.IID) : Git.Value?.GetInterfaceFromGlobal(_cookie, IUnknownVftbl.IID);
+        public IObjectReference Get() => _cookie == IntPtr.Zero ? ABI.WinRT.Interop.IAgileReferenceMethods.Resolve(_agileReference, IUnknownVftbl.IID) : Git.GetInterfaceFromGlobal(_cookie, IUnknownVftbl.IID);
```

Comment:
I had dropped it by accident, but looking at the code again I don't think that was correct. That factory code could never return `null`, because it had an `AsType<T>()` call at the end, which would just throw if the input was `null`. Not to mention that `CoCreateInstance` itself can't return `null` at all in case of success. So I did two things:
- Removed that `AsType` and used the explicit constructor to avoid reflection there
- Removed the `null` checks there antirely